sites/fast-component-explorer/app/explorer.props.ts

```diff
@@ -56,6 +57,11 @@ export interface ExplorerState {
      */
     direction: Direction;
 
+    /**
+     * The theme
```

Comment:
Is there only one theme?  Is this theme special. Is this a Standard Luminance theme among others?

Reply:
Currently there is only dark theme and light theme represented by the enum `StandardLuminance` which includes `StandardLumunance.DarkMode` and `StandardLuminance.LightMode`. It is conceivable that this will change in the future to have more, but there are no current plans for that.